oteapi/models/triplestoreconfig.py

```diff
@@ -26,11 +32,21 @@ class TripleStoreConfig(AttrDict):
         ...,
         description="AllegroGraph port number.",
     )
-    agraphUser: str = Field(
```

Comment:
For backwards compatibility, would it make sense to recreate the `SecretConfig.user` attribute and use the `agraphUser` alias for it? Or maybe it's just silly? I'm not sure how many actually uses it, nor how the attributes here are used elsewhere in general.

Reply:
I personally wonder why the `TripleStoreConfig` and `Triplestore`-strategy itself are so tightly restricted to agraph. I think it rather makes sense to focus more on the oteapi/tripper interfacing through ontokb in order to make a generic triplestore interface. Once this is provided, this `Triplestore`-strategy might be confusing for a potential oteapi-user/developer in the end.

An alias for `User` in the `TripleStoreConfig`would at least consistency provide a consistency in the attribute-names. I personally think that an `agraph`-prefix make the attributes quite long and it is kind of obvious that agraph is used as backend, when explained in the strategy-description

Reply:
I agree with everything here. I'll make an issue to update these classes and to point them towards the new implementations we have.